app/controllers/api/base_controller/parameters.rb

```diff
@@ -161,6 +161,8 @@ def sort_params(klass)
         options = String(params['sort_options']).split(",")
         params['sort_by'].split(",").zip(orders).collect do |attr, order|
           if klass.attribute_method?(attr) || klass.method_defined?(attr) || attr == klass.primary_key
+            raise BadRequestError,
+                  "#{klass.name} cannot be sorted by #{attr}" unless klass.attribute_supported_by_sql?(attr)
```

Comment:
Just wondering if `attribute_supported_by_sql?` implies the condition on 163? Could this be simplified?